src/index.html

```diff
@@ -18,8 +18,17 @@
             <button type="button" class="header__button-outline button-outline">Регистрация</button>
         </div>
         <div class="header__mobile-button">
-          <button class="header__button-hamburger"></button>
-      </div>
+          <button class="header__button-hamburger">
+            <span class="header__button-cross"></span>
+            <nav class="header__mobile-menu">
```

Comment:
??? what is that? Please read the docs
https://developer.mozilla.org/ru/docs/Web/HTML/Element/button#%D0%BE%D0%BF%D0%B8%D1%81%D0%B0%D0%BD%D0%B8%D0%B5
The button only supports text content